client/app/telecom/telephony/alias/special/telecom-telephony-alias-special.html

```diff
@@ -0,0 +1,47 @@
+<div class="telephony-group-alias-special">
+
+    <div class="text-center" data-ng-if="AliasSpecialCtrl.isLoading">
+        <oui-spinner></oui-spinner>
+    </div>
+
+    <div class="widget-presentation"
+         data-ng-if="!AliasSpecialCtrl.isLoading">
+        <div class="alert alert-info"
```

Comment:
You could use `<oui-message>`?

Reply:
In fact, this message is useless => it will be deleted